modules/ROOT/pages/deployment/storage/general-considerations.adoc

```diff
@@ -0,0 +1,173 @@
+** Though not required, the data path for services mentioned in xref:deployment/general/general-info.adoc#base-data-directory[Services That Can Deviate from the Base Data Path] can be deviated and individually configured. Using an additional partition/moint point is a possible way to achieve this.
+
+The following table describes where data should be stored when using Infinite Scale in production:
+
+{empty} +
+
+[role=center,width=70%,cols="40%,45%,60%",options="header"]
+|===
+| Envorinment Variable
+| Target
+| Space requirement
+
+| `OCIS_CONFIG_DIR`
+| Default (local)
+| Very low
+
+| `OCIS_BASE_DATA_PATH`
+| Partition
+| Medium
+|===
+
+== NFS
+
+Using NFS is benefical because the instance components are separated from the storage and can scale therefore independently. Such a setup is used when you plan a production environment with many users. As the same rules apply described in the topics of xref:local-storage[Local Storage], you mount instead using partitions NFS mount points. In addition, the Infinite Scale config is also stored on NFS using the xref:deployment/general/general-info.adoc#default-paths[OCIS_CONFIG_DIR] environment variable as this is a requirement for xref:deployment/container/orchestration/orchestration.adoc[orchestrated environments]. For details on NFS see the xref:deployment/storage/nfs.adoc[Network File System] documentation.
+
+{empty} +
+
+[role=center,width=70%,cols="40%,35%,80%",options="header"]
+|===
+| Envorinment Variable
+| Target
+| Space requirement
+
+| `OCIS_CONFIG_DIR`
+| NFS_1
+| Very low
+
+| `OCIS_BASE_DATA_PATH`
+| NFS_2
+| High
+
+| Seach and Thumbnails
+| NFS_3
+| Medium, to be monitored
+|===
+
+== S3
+
+S3 to store blobs is typically used by large enterprises and hosters, though it can fit for medium enterprises too. Data distribution and separation is a bit different compared to a pure POSIX backend. For details on S3 see the xref:deployment/storage/s3.adoc[S3] documentation. Also see the xref:deployment/general/general-info.adoc#using-s3-for-blobs[Using S3 for Blobs] description:
+
+* POSIX storage, usually NFS.
+** Metadata
+** Data for search and/or thumbnails
+** Other data
+
+* S3 for blobs
+
+With S3, data will be distributed over different storages and mounts based on their use case. With such a setup, the system can scale according the needs of large enterprises.
+
+{empty} +
+
+[role=center,width=90%,cols="40%,45%,80%",options="header"]
+|===
+| Envorinment Variable
+| Target
+| Space requirement
+
+| `OCIS_CONFIG_DIR`
+| NFS_1
+| Very low
+
+| `OCIS_BASE_DATA_PATH`
+| NFS_2
+| Medium, to be monitored +
+Alternatively low if NFS_4
+
+| Seach and Thumbnails
+| NFS_3
+| Medium, to be monitored
+
+| `STORAGE_USERS_S3NG_ROOT`
+| `OCIS_BASE_DATA_PATH` +
+Alternatively NFS_4
+| Medium, to be monitored
+
+| S3 specific settings
+| S3
+| High
+|===
+
+== Resource Optimisation
```

Comment:
ToDo: review this section...

Reply:
Will do with an independent PR to not block that one.